docs/endpoints/posts.md

```diff
@@ -14,6 +14,8 @@ Only admins and post owners will have `tags`, `source`, `remote_addr` (which wil
 
 Anonymous requests will only return accepted posts. Logged-in users can see accepted posts & any of their own pending or rejected posts. Staff can see anything!
 
+The `northstar_id` field will only be returned for the post owner.
```

Comment:
This is only true if the action is anonymous, right? Can we reword this to:

> If the post's action is marked as "anonymous", the `northstar_id` field will only be returned for the owner.

Reply:
ah shoot yes, I was going too fast yesterday! Updated :)